node/src/validator/mod.rs

```diff
@@ -67,10 +68,26 @@ pub struct Validator<N: Network, C: ConsensusStorage<N>> {
     handles: Arc<RwLock<Vec<JoinHandle<()>>>>,
     /// The shutdown signal.
     shutdown: Arc<AtomicBool>,
+    /// The primary keypair of the node exposed here for handshaking purposes.
+    primary_keypair: Arc<BLS12381KeyPair>,
+    /// Current consensus committee, might need to be mutable for dynamic committees.
+    committee: Committee,
     /// The running BFT consensus instance.
     bft: Arc<OnceCell<RunningConsensusInstance<BftExecutionState<N, C>>>>,
+
+    dev: Option<u16>,
 }
 
+// Validator Quorum:
+//
+// Before starting consensus, we need to ensure that the validator mesh is full formed. In order to
```

Comment:
fully

Reply:
Indeed—left over from me working this out, I'll remove this 👍